src/main.js

```diff
@@ -1 +1,30 @@
-console.log('Hello, world!');
+// console.log('Hello, world!');
+// const total = document.querySelector('.form__total');
+// console.log(total);
+// // total.textContent = '';
+// const radio = document.querySelectorAll('input[type="radio"]');
+// console.log(radio);
+// // console.log(radio.checked === true);
+// function radioPayment() {
+//   if (radio.checked === true) {
+//     radio.textContent = '$300';
+//     return true;
+//   }
+// }
+// radio.forEach(radio => radio.addEventListener('change', radioPayment));
+// // radio.forEa.addEventListener('click', radioPayment());
+const amount = document.querySelector('.form__total span');
+console.log(amount);
+const inputs = Array.from(document.querySelectorAll('input[type="radio"]'));
```

Comment:
Writing `querySelector` for forms is a pain! So here's a tip:

You can actually just grab the whole `<form>` tag and use the `name=" "` attributes of the containing form elements: 

```javascript
// Test these in devtools console tab
const form = document.querySelector('.form');

// This grabs the radio buttons named plan
form.plan
form.plan.value // Or its selected value

// This grabs the submit button
form.submit

// This grabs the email
form.email
```